desktop/extensions.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Extensions
+keywords: Extensions, Docker Desktop, Linux, Mac, Windows
+title: Extensions
+toc_min: 1
+toc_max: 2
+---
+
+Docker Extensions lets you use third-party tools within Docker Desktop to extend its functionality. There is no limit to the number of extensions you can install.
+
+Docker Community members and partners can use our [SDK](https://docker-desktop-extensions.netlify.app/){:target="_blank" rel="noopener" class="_"}  to create new extensions. 
```

Comment:
The link currently points to the Netlify site. @gtardif Wondering if we can also have the SDK docs ready when the user manual goes live?

Reply:
Currently reviewing the SDK docs now. They won't be ready to be released today along with the user manual. Was going to create a separate ticket to update with the new link when the SDK docs are ready.